CHANGELOG.md

```diff
@@ -1,18 +1,20 @@
 # Changelog
 
-### 0.3.0
+### 1.0.0
```

Comment:
Let's hold off on the 1.0.0 version for this change. We can do a dedicated PR for this. Instead let's just use 0.3.0.